libs/metrics/icu_headroom.py

```diff
@@ -1,229 +0,0 @@
-from typing import Union, Optional, Dict, Any, Tuple
-import numpy as np
-import pandas as pd
-
-from covidactnow.datapublic.common_fields import CommonFields
-from libs import series_utils
-from api import can_api_v2_definition
-
-
-NonCovidPatientsMethod = can_api_v2_definition.NonCovidPatientsMethod
-CovidPatientsMethod = can_api_v2_definition.CovidPatientsMethod
-
-
-# Default utilization to use (before decomp) if there isn't a location-specific
-# value in the API.
-DEFAULT_ICU_UTILIZATION = 0.75
-
-DEFAULT_ICU_DECOMP = 0.21
-
-NJ_CORRECTION = 0.176 - 0.57 + DEFAULT_ICU_DECOMP
```

Comment:
Thank goodness for the HHS hospital dataset. I had forgotten that we had be doing these bespoke corrections.

Reply:
I owe Kevin a beer.